web/src/infra/lobby/NewRoomCard.module.css

```diff
@@ -0,0 +1,24 @@
+.CardWrapper {
+    width: 219px;
+    min-width: 219px;
```

Comment:
Probably we don't need a min-width considering we already have a width.

Reply:
We do, I thought the same but somehow the card collapse on mobile if we dont have mind width.